### PR TITLE
render glossary as a typst term list

### LINKED ISSUE #208: Render glossary as a typst term list
Refactor how the glossary is rendered so it uses the typst native `terms()`: https://typst.app/docs/reference/model/terms/

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**
closes #208﻿

### CODE REVIEW

File: lib/glossary.typ [hunk 1]
```diff
@@ -41,7 +41,12 @@
...
     #heading(title) #label("glossary")
   ]
-  for (word, info) in data {
-    [/ #word: #info.definition\ → #if "link" in info {link(info.link)}]
-  }
-}
\ No newline at end of file
+  
+
+  (terms(..data.pairs().map(((word,info)) => terms.item(word, [#info.definition\ → #if "link" in info {link(info.link)}]))))
```
Comment (dyc3) [inline lib/glossary.typ:46]: nit: This appears to have an extra set of parentheses.

File: lib/glossary.typ [hunk 2]
```diff
@@ -41,7 +41,12 @@
   if data.len() > 0 [
     #heading(title) #label("glossary")
   ]
-  for (word, info) in data {
-    [/ #word: #info.definition\ → #if "link" in info {link(info.link)}]
-  }
-}
\ No newline at end of file
+  
+
```
Comment (dyc3) [inline lib/glossary.typ:45]: nit: remove extra newlines

File: lib/glossary.typ [hunk 3]
```diff
@@ -41,7 +41,12 @@
...
+  (terms(..data.pairs().map(((word,info)) => terms.item(word, [#info.definition\ → #if "link" in info {link(info.link)}]))))
+
+}
+
+
+
+
```
Comment (dyc3) [inline lib/glossary.typ:52]: nit: remove extra newlines, leave only one newline at the very end

**Review by dyc3 — COMMENTED** [review]
Looks good. We're gonna hold off on merging this for now until we've cleared everything up regarding the team switch with Darian.

File: lib/glossary.typ (hunk 3 above)
Comment (Victor-M-Giraldo) [reply lib/glossary.typ:52]: I'll fix these. I didn't reference the PR in the commit message. Should I just mess with the ```changelog.csv``` file so things don't look messy?

Comment (dyc3) [reply lib/glossary.typ]: `changelog.csv` is auto generated from the commit history. When a PR gets merged, it'll get squashed into a single commit, and it'll reference the PR in the message. No action needed.

**Review by cjrkoa — APPROVED** [review]

**Pull request merged**